cfme/configure/configuration/__init__.py

```diff
@@ -289,12 +247,55 @@ def __exit__(self, type, value, traceback):
         self.delete()
 
 
-class HostAnalysisProfile(AnalysisProfile):
-    CREATE_LOC = "host_analysis_profile_add"
+@navigator.register(AnalysisProfile, 'All')
+class AnalysisProfileAll(CFMENavigateStep):
+    prerequisite = NavigateToObject(Server, 'Configuration')
 
+    def step(self):
+        server_region = store.current_appliance.server_region_string()
+        self.parent_view.accordions.settings.tree.click_path((server_region, "Analysis Profiles"))
 
-class VMAnalysisProfile(AnalysisProfile):
-    CREATE_LOC = "vm_analysis_profile_add"
+
+@navigator.register(AnalysisProfile, 'Add')
+class AnalysisProfileAdd(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        tb.select("Configuration", "Add {type} Analysis Profile".format(type=self.obj.profile_type))
+
+
+@navigator.register(AnalysisProfile, 'Details')
+class AnalysisProfileDetails(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        server_region = store.current_appliance.server_region_string()
+        self.parent_view.accordions.settings.tree.click_path((server_region,
+                                                              "Analysis Profiles", str(self)))
+
+
+@navigator.register(AnalysisProfile, 'Edit')
+class AnalysisProfileEdit(CFMENavigateStep):
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        tb.select("Configuration", "Edit this Analysis Profile")
+
+
+@navigator.register(AnalysisProfile, 'Delete')
```

Comment:
Actually, you're right those aren't needed. As far as we have Add, Edit, etc destination, I think it would be good to have Delete as well. Moreover, Delete has alert that also can be considered as a destination :).
I can remove them if you want.